### DISCUSSION TITLE
Is there a way to disable/hide tags inside the dropdown button when there are several selected items?

### DISCUSSION THREAD

**Discussion by rezasurin** [azeezat/react-native-select#63] [Q&A]
Thank you for creating this package, it's very helpful!
Anyway, I want to hide tags or change the tags to a counter of selected items. Is there any suggestion or a way to do that?

**Comment by azeezat** [comment]
@rezasurin If I understand correctly, you want to customize the appearance of the dropdown menu when multiple items are selected?
